tests/chocolatey-tests/commands/testpackages/chocolatey-dummy-package/1.0.0/chocolatey-dummy-package.nuspec

```diff
@@ -12,7 +12,7 @@
     <summary>This is a purposely empty dummy package for testing choco push functionality</summary>
     <description>This is a purposely empty dummy package for testing choco push functionality</description>
     <dependencies>
-      <dependency id="chocolatey" />
+      <dependency id="chocolatey" version="1.0.1" />

```

Comment:
Again we should probably set the dependency range to be the minimum version of `chocolatey` that is available, not sure which version this would be but it definitely would be in a pre-v1 range at least.